### PR TITLE
feat: add custom command option

### PR DESCRIPTION
**Pull request by rickstaa**
This commit gives users the ability to supply custom commands that are executed after the process has finished. 

> **Warning**
> We, however, have to check if this is safe. I tried executing sudo commands but as long as tmux is not run as sudo this fails. @seekstar what do you think?

### CODE REVIEW

File: scripts/notify.sh [hunk 1]
```diff
@@ -9,6 +9,8 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${CURRENT_DIR}/helpers.sh"
 source "${CURRENT_DIR}/variables.sh"
 
+custom_command="echo 'hello world'"
```
Comment (seekstar) [inline scripts/notify.sh:12]: It looks like you forget to delete this line.

File: README.md [hunk 2]
```diff
@@ -110,6 +110,16 @@ The Tmux notify script uses your shell prompt suffix to check whether a command
 
 Feel free to open [a pull](https://github.com/rickstaa/tmux-notify/pulls) request or [issue](https://github.com/rickstaa/tmux-notify/issues) if you think your shell prompt suffix should be included by default.
 
+### Execute custom notification commands
+
+You can execute a custom command after a process has finished by putting `set -g @tnotify-custom-cmd 'your custom command here"` in the `.tmux.conf` file. The custom command is executed in the pane where the process has finished. If you want to execute multiple commands, you can also put them in a bash script and execute this script (i.e. `set -g @tnotify-custom-cmd 'bash /path/to/script.sh"`).
```
Comment (seekstar) [inline README.md:115]: `'your custom command here"`
The quote marks do not match.

**Comment by seekstar** [comment]
I have difficulties in using the new feature. I cloned the repo and checkout to `add_custom_command_option`. Then I modify `~/.tmux.conf` to use the local plugin:

```shell
# List of plugins
set -g @plugin 'tmux-plugins/tpm'
#set -g @plugin 'ChanderG/tmux-notify'
set -g @tnotify-custom-cmd "echo \\\\ > /tmp/test"
run-shell ~/git/others/tmux-notify/tnotify.tmux
set -g @tnotify-custom-cmd "echo \\\\ > /tmp/test"

# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm'
```

The original `tmux-notify` plugin has been removed, and `~/.tmux/plugins` has only `tpm`.

Then I exit all `tmux` sessions and then create a new one, run `top` there, enable monitoring, then exit `top`. Then a while later there is a notification on the desktop as usual, but `/tmp/test` does not exist, which indicates that the custom command is not executed. Are there any logs I can check?

**Comment by rickstaa** [comment]
> I have difficulties in using the new feature. I cloned the repo and checkout to `add_custom_command_option`. Then I modify `~/.tmux.conf` to use the local plugin:
> 
> ```shell
> # List of plugins
> set -g @plugin 'tmux-plugins/tpm'
> #set -g @plugin 'ChanderG/tmux-notify'
> set -g @tnotify-custom-cmd "echo \\\\ > /tmp/test"
> run-shell ~/git/others/tmux-notify/tnotify.tmux
> set -g @tnotify-custom-cmd "echo \\\\ > /tmp/test"
> 
> # Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
> run '~/.tmux/plugins/tpm/tpm'
> ```
> 
> The original `tmux-notify` plugin has been removed, and `~/.tmux/plugins` has only `tpm`.
> 
> Then I exit all `tmux` sessions and then create a new one, run `top` there, enable monitoring, then exit `top`. Then a while later there is a notification on the desktop as usual, but `/tmp/test` does not exist, which indicates that the custom command is not executed. Are there any logs I can check?

@seekstar thanks for helping me implement this feature. I haven't tried the method above, but I would include the branch as `set -g @plugin 'rickstaa/tmux-notify#add_custom_command_option'`. If you use this method, you can specify the custom command as:

```bash
set -g @tnotify-custom-cmd 'echo hello'
```

There appeared to be a bug which I fixed in 6f965c291358c15ed1d38ba18d10a630325ee75f.

**Comment by seekstar** [comment]
Thank you very much for implementing this feature. It works perfectly now.

**Pull request merged**